ucm2/Intel/sof-hda-dsp/HiFi-analog-sof.conf

```diff
@@ -0,0 +1,492 @@
+# Generic HDA devices for analog I/O
+
+Define.FrontMicName ""
+Define.FrontMicJack ""
+Define.DeviceFrontMic ""
+Define.LineDevice ""
+Define.hpvol "Headphone"
+Define.hpjack "Headphone Jack"
+Define.loctl "Line"
+Define.lovol "Line"
+
+# IPC version
+# TODO need to duplicate below definions to True and False since Define after
+# this if is not seeing variable SOFIPCVer.
+If.SOFIPCVer {
+	Condition {
+		Type ControlExists
+		Control "name='Post Mixer Analog Playback IIR Eq bytes'"
+	}
+	True.Define {
+	      SOFIPCVer "ipc4"
+	      PostMixerAnalogPlaybackIIRBytes "Post Mixer Analog Playback IIR Eq bytes"
+	      PostMixerAnalogPlaybackFIRBytes "Post Mixer Analog Playback FIR Eq bytes"
+	      PostMixerAnalogPlaybackDRCBytes "Post Mixer Analog Playback DRC bytes"
+	      PostMixerAnalogPlaybackDRCSwitch "Post Mixer Analog Playback DRC switch"
+	      SpeakerIirBlob "${ConfTopDir}/sof/${var:SOFIPCVer}/eq_iir/highpass_100hz_0db_48khz.blob"
```

Comment:
hard-coding the path makes it difficult to override with a different blob.

I don't have a solution but maybe there should be a separate variable for the path prefix that can use an environment variable or something.

Reply:
A variable that holds ${ConfTopDir}/sof/${var:SOFIPCVer} would be simple and clean up the code a bit. These are the defaults so I don't expect them to change much. For customize that I'd need help from experts.

Reply:
Yep, it's possible to access environment variables, with ${env:<str>}. @plbossart Do you have suggestions for it and should it override ${ConfTopDir}/sof or even the ${var:SOFIPCVer} too?

Reply:
I was only referring to a path change (similar to what we do for firmware/topology).

the IPC version cannot be changed IMHO, it's a given on a platform. On recent kernels we expose it with debugfs, but that requires root access.